astrapy/db.py

```diff
@@ -694,6 +697,51 @@ def insert_many(
 
         return response
 
+    def batched_concurrent_insert_many(
```

Comment:
It seems that `batched` is redundant with `many`.
Maybe `concurrent_insert_many` would be enough ?

Reply:
I'll go with `chunked` based on the way we typically describe it, good suggestion, thank you!